app/models/dialog.rb

```diff
@@ -140,11 +141,11 @@ def field(name)
   end
 
   def content(target = nil, resource_action = nil, all_attributes = false)
-    return DialogSerializer.new.serialize(Array[self], all_attributes) if target.nil? && resource_action.nil?
+    return DialogSerializer.new.serialize([self], all_attributes) if target.nil? && resource_action.nil?
```

Comment:
Yea, would feel more comfortable with `Array()` here

Reply:
I'll back this one out, and we can take these case-by-case ... if I recall there's maybe less than a dozen of these